Code_Exercises/Exercise_02_Hello_World/source.cpp

```diff
@@ -12,5 +12,21 @@
 #include <catch2/catch.hpp>
 
 TEST_CASE("hello_world", "hello_world_source") {
+  
+  // Create a default SYCL queue for submitting events using sycl::queue{}
+  // the slides use a GPU queue, but we recommend default queue as this 
+  // will select the best device for you
```

Comment:
Minor nit, but I would maybe say the SYCL runtime will choose a device for you rather than the best device for you, as it depends on what is available and what the user is looking for.